pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -206,7 +214,24 @@ func (p *PollerWorker) Poll(execAgentCommands agentCommandsExecutor, closeChan c
 		case <-ticker.C:
 			execAgentCommands(p.AgentCommands)
 		}
+		if pollInterval < pollStableInterval {
+			pollInterval = incrementPollInterval(pollInterval, pollStableInterval)
+			ticker = replaceTicker(ticker, pollInterval)
+		}
```

Comment:
maybe move to the ticker case ?
it will be easier to see it happens just there

Reply:
I do not see why there is a need to indent here. I find it more correct to say that the action is performed after the `select` and place it directly in the for-loop rather in the `select` itself.

Reply:
as its connected to the case of the timer i think its better there, up to you